ext/dom/tests/DOM4_DOMNode_ElementSiblings.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+DOMNode: Element Siblings
+--SKIPIF--
+<?php require_once('skipif.inc'); ?>
+--FILE--
+<?php
+require_once("dom_test.inc");
+
+$dom = new DOMDocument;
+$dom->loadXML('<test>foo<bar>FirstElement</bar><bar>LastElement</bar>bar</test>');
+if(!$dom) {
```

Comment:
Unless DOM is doing something super magical (does it?) these conditions can't ever be false, right? It'll still be a DOMDocument object.

Reply:
This is copied from other tests :-D Sorry for being lazy and not cleaning it up :-)